lib/cn/csched_sp3.c

```diff
@@ -1506,6 +1506,7 @@ sp3_job_serialize(struct sts_job *const job, void *const arg)
     ulong tm;
     char tmbuf[32];
     merr_t err = 0;
+    bool bad = false;
```

Comment:
nit: you could initialize 'bad' in line 1528 and get rid of this.

Reply:
This is what you mean?

```
bad = false | !cJSON_AddNumberToObject(node, "cnid", w->cw_tree->cnid);
```

Reply:
or maybe

```
bad = false;
bad = false | !cJSON_AddNumberToObject(node, "cnid", w->cw_tree->cnid);
```

Reply:
The last one, just lose the 'bad = false;'